ganga/GangaCore/Lib/BackendScriptTemplate.py.template

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
@egede  This shebang line in principle is neutral to the version of the python. Ref: [link](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take), [link](https://mail.python.org/pipermail/tutor/2007-June/054816.html) I have taken care to make the code both python2 and python3 compatible.